packages/ui/v2/modules/List.tsx

```diff
@@ -3,18 +3,31 @@ import { createElement } from "react";
 
 import classNames from "@calcom/lib/classNames";
 
-export function List(props: JSX.IntrinsicElements["ul"]) {
+export type ListProps = {
+  roundContainer?: boolean;
+} & JSX.IntrinsicElements["ul"];
+
+export function List(props: ListProps) {
   return (
-    <ul {...props} className={classNames("-mx-4 rounded-sm sm:mx-0 sm:overflow-hidden", props.className)}>
+    <ul
+      {...props}
+      className={classNames(
+        "-mx-4 rounded-sm sm:mx-0 sm:overflow-hidden ",
+        // Add rounded top and bottome if roundContainer is true
+        props.roundContainer && "[&>*:first-child]:rounded-t-md [&>*:last-child]:rounded-b-md ",
```

Comment:
It would be ideal if we could pass `[&_a]:rounded-none` here but for some reason specificity doesn't like it and ignores.